lms/djangoapps/certificates/models.py

```diff
@@ -309,6 +315,61 @@ def __unicode__(self):
                ("regenerated" if self.is_regeneration else "generated", self.generated_by, self.created, self.course_id)
 
 
+class CertificateInvalidation(TimeStampedModel):
+    """
+    Model for storing Certificate Invalidation.
+    """
+    generated_certificate = models.ForeignKey(GeneratedCertificate)
+    invalidated_by = models.ForeignKey(User)
+    notes = models.TextField(default=None, null=True)
+    active = models.BooleanField(default=True)
```

Comment:
@saleem-latif can you please tell me what is the purpose of keeping the `active` flag `True` ?